### PR TITLE
Update wizard

### PR DESCRIPTION
**Pull request by martialblog**
- Uses newer showoff version
- Creates artifacts with the user's permissions
- Remove showoff sections markers from PDF
- Add option to select PDF format (A5, A4)

This PR also adds the LICENCE to the handout

### CODE REVIEW

File: global/Pre/Title/04_Licence.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+!SLIDE printonly
+
+# Licence
+
+This work is licensed under a Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0) License.
```
Comment (dgoetz) [inline global/Pre/Title/04_Licence.md]: Should be CC BY-SA 4.0

File: global/wizard.sh [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
+    $RUNTIME rm -f $CNAME 2> /dev/null
   fi
 
-  $DOCKER run \
+  $RUNTIME run \
     -it \
     --name=$CNAME \
     --rm \
     -p 9090:9090 \
+    --user $(id -u) \
```
Comment (dgoetz) [inline global/wizard.sh:28]: Requires also `--userns=keep-id` if using podman otherwise it will fail.

File: global/wizard.sh [hunk 3]
```diff
@@ -96,6 +108,27 @@ esac
 
 setlayout $LAYOUT
 
+echo -e "\n### FORMAT ###"
+
+echo -e "
+   [1] A5
+   [2] A4\n"
```
Comment (dgoetz) [inline global/wizard.sh:121]: A4 is generated with an asymmetric page layout for exercise and solutions. Handout would need even more adjustments with the fist page looking bad and pagebreaks and pictures needing adjustment. So I would not use it for now, but if it is meant as a preparation I am not against it.

**Review by dgoetz — CHANGES_REQUESTED** [review]
Thanks, but some changes are needed (and perhaps also ported to the original one).

File: global/Pre/Title/04_Licence.md (hunk 1 above)
Comment (martialblog) [reply global/Pre/Title/04_Licence.md]: Copypasta... fixed it.

File: global/wizard.sh (hunk 2 above)
Comment (martialblog) [reply global/wizard.sh:28]: Thanks for the hint. Fixed it.

File: global/wizard.sh (hunk 3 above)
Comment (martialblog) [reply global/wizard.sh:121]: We recently added it to the wizard I think more as a future option. The default remains A5 anyways.

File: global/wizard.sh [hunk 4]
```diff
@@ -1,26 +1,27 @@
...
   fi
 
-  $DOCKER run \
+  $RUNTIME run \
     -it \
     --name=$CNAME \
     --rm \
     -p 9090:9090 \
+    --user $(id -u) \
+    --userns=keep-id \
```
Comment (martialblog) [inline global/wizard.sh]: :thinking: needs to be optional for Docker

Comment (martialblog) [reply global/wizard.sh]: @dgoetz the latest version should work with Podman and Docker. Can you confirm that?

Comment (dgoetz) [reply global/wizard.sh]: It only works if you explicitly set the Environment variable $RUNTIME as the command -v enforces docker which works perfectly fine with an alias.  The alias is also fine if we switch the case statement to `case $($RUNTIME -v) in`. (tested with podman only which shows "podman version X.Y.Z", but docker has "Docker version X.Y.Z, build HASH" as compact version string so it should be fine)

Comment (martialblog) [reply global/wizard.sh]: I usually use it like this, which works in both cases

```
RUNTIME=$(command -v docker) ./global/wizard.sh

RUNTIME=$(command -v podman) ./global/wizard.sh
```

Comment (martialblog) [reply global/wizard.sh]: @dgoetz Is the current version OK you think?

Comment (martialblog) [reply global/wizard.sh]: Updated the Branch

**Review by dgoetz — APPROVED** [review]
LGTM, thanks!

**Pull request merged**